infrastructure/cli/pkg/cmd/create/helm.go

```diff
@@ -187,6 +187,7 @@ func (h *Helm) upsertAiryConfigMap() error {
 }
 
 func (h *Helm) cleanupJob() error {
+	return nil
```

Comment:
either this is a "typo" or I don't actually get what's going on here

Reply:
That was me testing the behavior 🙇